includes/admin/settings/display-settings.php

```diff
@@ -22,7 +22,13 @@
  */
 function edd_options_page() {
 	$active_tab = isset( $_GET['tab'] ) && array_key_exists( $_GET['tab'], edd_get_settings_tabs() ) ? $_GET['tab'] : 'general';
-
+	$sections = edd_get_settings_sections( $active_tab );
+	$key = 'main';
+	if ( is_array( $sections ) ) {
+		$key = key( $sections );
+	}
+	
+	$section = isset( $_GET['section'] ) && array_key_exists( $_GET['section'], edd_get_settings_sections( $active_tab ) ) ? $_GET['section'] : $key;	
```

Comment:
array_key_exists() expects parameter 2 to be array, boolean given
wp-content/plugins/easy-digital-downloads/includes/admin/settings/display-settings.php:31

Seeing this error on that `array_key_exists` call on the Extensions and Licenses tab.